### ISSUE TITLE
[Snackbar] Snackbar messages should have an accessibilityHint

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#4923]
Clients have no way to set the `accessibilityHint` of the Snackbar's message (the title label). The default `accessibilityHint`, "Double-tap to dismiss", should not be used since the gesture may be incorrect.

## Reproduction steps
In an MDCSnackbarMessage or MDCSnackbarMessageView, attempt to set an `accessibilityHint` value for the message.

## Expected behavior
MDCSnackbarMessageView uses its `accessibilityHint` for the `accessibilityHint` of its label, similar to how it uses `accessibilityLabel`.

## Actual behavior
In internal builds, the Snackbar message has no accessibility hint and there is no way to assign one.

## Platform (please complete the following information)
iPhone 4S/iOS 9.3.5

## Additional context
Internal build (development build via Xcode).
